R/theme_gtsummary_roche.R

```diff
@@ -17,7 +18,7 @@
 #' tbl_demographics(trial, by = trt, include = c(age, grade))
 #'
 #' reset_gtsummary_theme()
-theme_gtsummary_roche <- function(set_theme = TRUE, font_size = NULL) {
+theme_gtsummary_roche <- function(set_theme = TRUE, font_size = 8) {
   # start with the compact theme -----------------------------------------------
   lst_theme <- gtsummary::theme_gtsummary_compact(set_theme = FALSE, font_size = font_size)
```

Comment:
can we take these stylings out of gtsummary and add them explicitly here?

Reply:
I will try

Reply:
Wait, you mean all of them? I think it is not really necessary as they are overloaded by later calls. Or do you mean only the flextable ones?

Reply:
Just the ones we need will be sufficient. I prefer not to create an unnecessary dependency on the other theme